exetera/core/dataset.py

```diff
@@ -36,9 +36,10 @@ def __init__(self, session, dataset_path, mode, name):
         self._dataframes = dict()
 
         for group in self._file.keys():
-            h5group = self._file[group]
-            dataframe = edf.HDF5DataFrame(self, group, h5group=h5group)
-            self._dataframes[group] = dataframe
+            if group not in ('trash'):
```

Comment:
This will check that `group` is not in the string `trash`, did you mean `("trash",)`?